ccd-definition/CaseEventToComplexTypes/NotifyDefendantOfClaim-COS-nonprod.json

```diff
@@ -0,0 +1,180 @@
+[
+  {
+    "ID": "Claimant",
+    "CaseEventID": "NOTIFY_DEFENDANT_OF_CLAIM",
+    "CaseFieldID": "cosNotifyClaimDefendant1",
+    "ListElementCode": "cosDateOfServiceForDefendant",
+    "FieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY"
+  },
+  {
+    "ID": "Claimant",
+    "CaseEventID": "NOTIFY_DEFENDANT_OF_CLAIM",
+    "CaseFieldID": "cosNotifyClaimDefendant1",
+    "ListElementCode": "cosServedDocumentFiles",
+    "EventElementLabel": "What documents did you serve?",
+    "FieldDisplayOrder": 3,
+    "DisplayContext": "MANDATORY"
+  },
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
CaseType ID is usually set on caseEventtofieldtype file. is there any reason we have set it in complextype json file?